legacy/application/models/Show.php

```diff
@@ -169,6 +169,58 @@ public function setAutoPlaylistId($playlistid)
         $show->setDbAutoPlaylistId($playlistid);
     }
 
+    public function getHasOverrideIntroPlaylist()
+    {
+        $show = CcShowQuery::create()->findPK($this->_showId);
+
+        return $show->getDbOverrideIntroPlaylist();
+    }
```

Comment:
This feels expensive for a single get, I wonder if the show is not already prefilled, or maybe we should cache the show in a property.